src/main/java/io/neonbee/data/DataVerticle.java

```diff
@@ -183,10 +390,11 @@ public void start(Promise<Void> promise) {
             ResolutionRoutine routine;
             MultiMap headers = message.headers();
             try {
-                routine = message.body().getAction() == READ
-                        ? resolutionRoutineForStrategy(Optional.ofNullable(headers.get(RESOLUTION_STRATEGY_HEADER))
-                                .map(ResolutionStrategy::valueOf).orElse(RECURSIVE))
-                        : new ManipulationRoutine();
+                routine =
+                        message.body().getAction() == READ
+                                ? resolutionRoutineForStrategy(ofNullable(headers.get(RESOLUTION_STRATEGY_HEADER))
```

Comment:
Same here no default import of "of" methods please. (We should add that to our coding conventions in the docs folder please)